api/notifications.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+import (
+	"github.com/wtg/shuttletracker/model"
+	"net/http"
+	"strings"
+	"encoding/json"
+)
+
+// AdminMessageHandler handles the retrieval of the current administrator message
+func (api *API) AdminMessageHandler(w http.ResponseWriter, r *http.Request) {
+	message, err := api.db.GetCurrentMessage()
+
+	if err != nil {
+		http.Error(w,err.Error(), http.StatusInternalServerError)
+	}
+	WriteJSON(w, message)
+}
+
+// SetAdminMessage allows the user to set an alert message that will display to all users who visit the page
+func (api *API) SetAdminMessage(w http.ResponseWriter, r *http.Request){
+	message := model.AdminMessage{}
+	err := json.NewDecoder(r.Body).Decode(&message)
+
+	if err != nil {
+		http.Error(w,err.Error(), http.StatusInternalServerError)
+		return;
+	}
+	err = api.db.ClearMessage()
+	message.Message = strings.Replace(message.Message,"<script>","",-1)
+	message.Message = strings.Replace(message.Message,"</script>","",-1)
```

Comment:
Looking for `<script>` tags like this will not catch them if they have spaces. We could use a package to strip HTML tags except for `<a>`, `<i>`, etc, in addition to preventing the frontend from executing anything.